Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM public.ecr.aws/lambda/python:3.8
```

Comment:
would 3.10 be ok?

Reply:
Yeah I don't see any issues. Will test and push the change.

Reply:
Latest seems to be python3.9: https://docs.aws.amazon.com/lambda/latest/dg/python-image.html

Reply:
And it works just fine.

Reply:
Got it -- is there no public spec w.r.t. bases and do they all have to run using public.ecr.aws? py310 was released about a month ago, so it seems they have not yet gotten to releasing the lambda base for it.